<!DOCTYPE html>
<meta charset=utf-8>
<title><title><title><title>Uncaught exception following subtest</title></title></title></title>
<script src="/resources/testharness.js"></script>
<script src="/resources/testharnessreport.js"></script>
<script>
test(function() {});
throw new Error("error outside any setup or test");
</script>
